src/pages/RegistrationPage.js

```diff
@@ -83,15 +72,11 @@ const labelNames = id => {
     case 'emailConfirm':
       return <Trans>Confirm Email address</Trans>
     case 'accessibility':
-      return ''
+      return 'TODO: Input correct string'
```

Comment:
accessibility and privacy are the same, we should delete this case

Reply:
I removed Privacy and set the proper string, thanks for catching that!